### DISCUSSION TITLE
Development slow down in 2021 ?

### DISCUSSION THREAD

**Discussion by OlivierDelbeke** [carla-simulator/carla#4171] [General]
Hello,

We noticed a slow down in the number of commits since January 2021. What is the reason behind this ? I know it's a difficult question for open-source projects, but what are the plans for the future ? Do you have any kind of roadmap ? 

Thanks a lot !

**Comment by germanros1987** [comment]
Hi @OlivierDelbeke,

Development pace is never constant in software projects. 2020 was a year of bringing many new small features. 2021 is being a year of fixing a few but very critical large features, e.g. physics, determinism, the synchronous pipeline, etc.

The CARLA organization has strong plans for the present and future, and the roadmap is shared with our sponsors and partners. We typically don't make our roadmap fully available for the community (for reasons I am not going to discuss in this post), but we try to keep the community informed of what is coming. Let me provide you with a summary:

0.9.12: 
* Support for large maps (+10K sq mi)
* Updated physics API and deterministic behavior
* Enhanced parallel execution of raycasting sensors
* Support for Chrono vehicle physics (co-simulation)
* Full support for CarSim
* Upgrades in traffic scenarios and the leaderboard
* Upgrades in the ROS bridge
* New sets of vehicles and pedestrians (gen2).
* Suppor for UE4.26
* Optical flow ground truth

Later:
* Support for multi-gpu nodes
* Support for bi-directional lanes and left-side traffic
* Extended procedural generation tools to create large maps with materials, decals, etc.

Hope this information helps.

**Reply by OlivierDelbeke** [reply]
Thanks a lot for this very detailed answer. That's very good news !
